docs/presentation/Content/ECExpressions.md

```diff
@@ -30,3 +30,10 @@ The below table lists all of them:
 
 Additionally, [User Setting symbols](../Advanced/ECExpressions.md#symbols-in-global-context) are
 available as well.
+
+## Property Specification `isDisplayed` Attribute
+
+Property specification `isDisplayed` attribute can be an ECExpression which specifies whether particular property should be present in the result.
+
+Only [Ruleset variable symbols](../Advanced/ECExpressions.md#ruleset-variables-user-settings)
+are available for this ECExpression.
```

Comment:
```suggestion
## Property overrides

Some [property overrides](./PropertySpecification.md) support ECExpressions to determine the value. Only [Ruleset variable symbols](../Advanced/ECExpressions.md#ruleset-variables-user-settings)
are available in this ECExpression context.
```

The link in `PropertySpecification.md` will need to be adjusted to target this section after the rename.